arc_solvers/models/qa/multi_choice/qa_multi_choice_max_att.py

```diff
@@ -109,11 +111,7 @@ def __init__(self, vocab: Vocabulary,
 
         # question to choice attention
         att_question_to_choice_params = params.get("att_question_to_choice")
-        if "tensor_1_dim" in att_question_to_choice_params:
-            # automatically infer the input size
-            att_question_to_choice_params = update_params(att_question_to_choice_params,
```

Comment:
It looks like you removed the call to `update_params`, but the file implementing it, and all the magic in it for manipulating `Params`, is still there?